crates/sui-core/src/authority/authority_store.rs

```diff
@@ -1169,10 +1169,11 @@ impl AuthorityStore {
     /// executed locally on the validator but didn't make to the last checkpoint.
     /// The effects of the execution is reverted here.
     /// The following things are reverted:
-    /// 1. Certificate and effects are deleted.
-    /// 2. Latest parent_sync entries for each mutated object are deleted.
-    /// 3. All new object states are deleted.
-    /// 4. owner_index table change is reverted.
+    /// 1. Latest parent_sync entries for each mutated object are deleted.
+    /// 2. All new object states are deleted.
+    /// 3. owner_index table change is reverted.
+    /// NOTE: certificate and effects are not deleted, in case this transaction
```

Comment:
nit: transactions instead of certificates. Also might be worth explaining more details given this is pretty tricky. Also feel free to include more context of what we discusssed

Reply:
I was just keeping the same wording as before but happy to change it! Also fleshed out the comment as suggested